### DISCUSSION TITLE
Not able to Test the new MailBox Connector

### DISCUSSION THREAD

**Discussion by GunjanBR7** [microsoft/powercat-business-approvals-kit#250] [Q&A]
While all things are set using the Set up for new MailBox Connector - I am not able to Test the new connector, In Test Operation I see the Connections but In Operations (0) is showing. Please suggest if I am missing any step.
![Git Issue](https://github.com/microsoft/powercat-business-approvals-kit/assets/161579229/a212ba0a-be0d-44cb-8511-2ecd49b13423)

**Comment by girishuppal** [comment]
Hello @GunjanBR7 - Please have a look in to the video of Approvals kit https://youtu.be/N7mBRZ4F7tI?list=PL5xdZrvu1OhUIMH_Y3FAZwoyVqJOlGFHG  

Please follow all the steps with regards to creating App registration and then setting up custom connector. Thanks.
